sentinel-core/transport/command/http_command_center.cc

```diff
@@ -0,0 +1,70 @@
+#include "sentinel-core/transport/command/http_command_center.h"
+
+#include <future>
+#include <iostream>
+
+#include "sentinel-core/transport/command/http_command_utils.h"
+
+namespace Sentinel {
+namespace Transport {
+
+bool HttpCommandCenter::Start() {
+  if (http_server_) {
+    // log warn
+    return false;
+  }
+
+  http_server_.reset(new HttpServer(std::bind(&HttpCommandCenter::OnHttpRequest,
+                                              this, std::placeholders::_1)));
+
+  return http_server_->Start();
+}
+
+void HttpCommandCenter::Stop() {
+  if (http_server_) {
+    http_server_->Stop();
+  }
+}
+
+bool HttpCommandCenter::RegisterCommand(const std::string& command_name,
+                                        CommandHandlerPtr handler) {
+  if (command_name.size() == 0) {
+    return false;
+  }
+
+  if (handler_map_.find(command_name) != handler_map_.end()) {
+    // log warn
+    return false;
+  }
+
+  handler_map_[command_name] = handler;
+  return true;
+}
+
+void HttpCommandCenter::OnHttpRequest(struct evhttp_request* http_req) {
+  auto request = HttpCommandUtils::ParseHttpRequest(http_req);
+  auto target = request.GetMetadata(CommandRequest::kRequestTarget);
+
+  auto it = handler_map_.find(target);
+  if (it == handler_map_.end()) {
+    // log warn: can not find handler
```

Comment:
If the handler is absent, maybe we should also return a 400 response to indicate the error like this:

```
Unknown command `abc`
```